### PR TITLE
Use only the nickname for XEP-0392 color generation

### PR DESCRIPTION
**Pull request by horazont**
There have been complaints, and using only the nickname works much better in many scenarios.

### CODE REVIEW

**Comment by mathieui** [comment]
Thanks. The only issue I can see so far is that our own color changes when we /nick, while it should stay at the color fixed by the theme.

File: poezio/tabs/muctab.py [hunk 1]
```diff
@@ -650,32 +650,35 @@ def on_user_join(self, from_nick, affiliation, show, status, role, jid,
     def on_user_nick_change(self, presence, user, from_nick, from_room):
         new_nick = presence.xml.find('{%s}x/{%s}item' %
                                      (NS_MUC_USER, NS_MUC_USER)).attrib['nick']
+        old_color = user.color
+        user.change_nick(new_nick)
```
Comment (mathieui) [inline poezio/tabs/muctab.py]: This makes the condition of the next line always fail, since the nick has already changed

Comment (horazont) [reply poezio/tabs/muctab.py]: Ah, so that’s why it happens. Thanks. I may push a fix tomorrow.

**Comment by horazont** [comment]
@mathieui Should be fixed now.

**Pull request merged**